src/App.tsx

```diff
@@ -22,74 +23,126 @@ enum SortType {
 }
 
 type ReorderOptions = {
-  sortType: SortType,
-  isReversed: boolean,
+  sortType: SortType;
+  isReversed: boolean;
 };
```

Comment:
It would be very cool to create a separate type.ts file and put all the types/interfaces/enams and so on there, and use these types via import in the component file. This would provide us with greater code cleanliness.